assets/app/styles/_components.less

```diff
@@ -266,7 +296,7 @@
 .pod-block {
   .overview-pods {
     margin-right: 10px;
-    min-width: 190px;
+    min-width: 200px;
```

Comment:
Hm. I think with the new styles, it's better to keep them left-aligned. I'll undo that change in my autoscaling pull request.